hash/mac_hdflop.xml

```diff
@@ -5,125 +5,3347 @@ license:CC0
+		<publisher>MicroProse</publisher>
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk1.img" size="1474560" crc="51e78a07" sha1="bd38b4e1b4f057b4221f0eecf701fda9252b2c8d" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk2.img" size="1474560" crc="e68d61d7" sha1="04660d264db37f29cef2535c6e80fab721c895b0" />
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 3" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk3.img" size="1474560" crc="57b49a19" sha1="b718d9deb36652108938121e7eab514496fe3eaa" />
+			</dataarea>
+		</part>
+		<part name="flop4" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 4" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk4.img" size="1474560" crc="f0684c1a" sha1="f775747910baa887a2e53a6c2a97c3de90e2d135" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="clarisd10v3">
+		<description>ClarisDraw 1.0v3</description>
+		<year>1993</year>
+		<publisher>Claris</publisher>
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk01.img" size="1474560" crc="7e12bb1c" sha1="a8a84383b1d5db0e1a8793bb238de5a2779e81c1" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk02.img" size="1474560" crc="473e95af" sha1="6cd471d4425d96cb9923d4b43bed400e0e3fc5a1" />
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 3" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk03.img" size="1474560" crc="3b1f90dd" sha1="43803d3e21222d8b4e1a895c3ae204faa724dfb7" />
+			</dataarea>
+		</part>
+		<part name="flop4" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 4" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk04.img" size="1474560" crc="7c37142f" sha1="0afdb39d99e9d8f82b03c889192f27f1773af65b" />
+			</dataarea>
+		</part>
+		<part name="flop5" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 5" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk05.img" size="1474560" crc="8dafa209" sha1="5340297877b50076c6a7bd81d66b55960fefa3cb" />
+			</dataarea>
+		</part>
+		<part name="flop6" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 6" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk06.img" size="1474560" crc="a83aed12" sha1="c1991ee1f6712ea0b6372dd423e71a82c89f509e" />
+			</dataarea>
+		</part>
+	</software>
+
 	<!-- S/N: Z93350-001A -->
-	<software name="clarisw" supported="no">
+	<software name="clarisw">
```

Comment:
Can you put the version and language in the name?  ClarisWorks 1 and 2 are very different pieces of software.

Reply:
This was one of the few entries that were already there before, so I inadvertently skipped it. It should be consistent with the rest now.